projects/addon-doc/tokens/doc-icons.ts

```diff
@@ -12,12 +12,17 @@ export interface TuiDocIcons {
     readonly copy: string;
     readonly check: string;
     readonly link: string;
+    readonly externalLink: string;
```

Comment:
Should be optional to not have breaking change until v5

Reply:
fixed